### PR TITLE
Feature/wrk 70

### PR DESCRIPTION
**Pull request by Jos-Flock**

### CODE REVIEW

File: src/develop/kotlin/mocks/LoadData.kt [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
-)
+        private val userRepository: UserRepository
+) {
+    fun loadWhenEmpty(block: () -> Unit) {
+
+        if(load == null){
+            load = userRepository.count() == 0L
+        }
+
+        if (load == true) {
```
Comment (tkgroot) [inline src/develop/kotlin/mocks/LoadData.kt:18]: whitespace betweeen closing paranthesis and bracket ') {'
is the code formatted correctly?

File: src/develop/kotlin/mocks/LoadData.kt [hunk 2]
```diff
@@ -1,11 +1,26 @@
...
+            block()
+        }
+    }
+
+    companion object{
```
Comment (tkgroot) [inline src/develop/kotlin/mocks/LoadData.kt:23]: again something about code formatting. opening bracket without whitespacing or with?

File: src/main/react/components/StatusMenu.js [hunk 3]
```diff
@@ -1,88 +1,125 @@
...
-    backgroundColor: "unset",
-  },
-  buttonApproved: {
-    backgroundColor: theme.palette.success[500],
-  },
-  buttonRejected: {
-    backgroundColor: theme.palette.error[500],
-  },
+    buttonRequested: {
+        backgroundColor: "unset",
```
Comment (tkgroot) [inline src/main/react/components/StatusMenu.js:11]: four whitespaces as indentation? Do you use the right prettier configurations?

**Comment by tkgroot** [comment]
Just another small thing. Do we have commit naming conventions? I see that all commits start with a jira ticket number.
Did we follow conventional-commits as standard?

**Comment by Jos-Flock** [comment]
Staat al in Master

**Pull request closed without merging**